fog-kubevirt.gemspec

```diff
@@ -6,6 +6,9 @@ require 'fog/kubevirt/version'
 Gem::Specification.new do |spec|
   # Dynamically create the authors information {name => e-mail}
   authors_hash = Hash[`git log --no-merges --reverse --format='%an,%ae'`.split("\n").uniq.collect {|i| i.split(",")}]
+  # Fog maintainers
+  authors_hash["Wesley Beary"]                 = "geemus@gmail.com"
+  authors_hash["Paulo Henrique Lopes Ribeiro"] = "plribeiro3000@gmail.com"
```

Comment:
You can leave me out here.

Wesley is the only author of `fog`.

Reply:
He asked you to add you here but if you like I can remove you.

Reply:
Only in the actual `rubygem` that you are going to release to [rubygems](rubygems.org). =)